data/json/speech.json

```
@@ -363,55 +363,55 @@
     "type": "speech",
     "speaker": [ "mon_mi_go", "mon_mi_go_slaver", "mon_mi_go_myrmidon" ],
     "sound": "\"Kill them all and let God sort them out!\"",
-    "volume": 40
+    "volume": 32
   },
   {
     "type": "speech",
     "speaker": [ "mon_mi_go", "mon_mi_go_slaver" ],
     "sound": "\"I watched a snail crawl along the edge of a straight razor.\"",
-    "volume": 40
+    "volume": 20
   },
   {
     "type": "speech",
     "speaker": [ "mon_mi_go", "mon_mi_go_slaver" ],
     "sound": "\"I've seen horrors, horrors that you've seen.\"",
-    "volume": 40
+    "volume": 20
   },
   {
     "type": "speech",
     "speaker": [ "mon_mi_go", "mon_mi_go_slaver" ],
     "sound": "\"I love the smell of napalm in the morning.\"",
-    "volume": 40
+    "volume": 20
   },
   {
     "type": "speech",
     "speaker": [ "mon_mi_go", "mon_mi_go_slaver", "mon_mi_go_myrmidon" ],
     "sound": "\"This is the way the fuckin' world ends.\"",
-    "volume": 40
+    "volume": 20
   },
   {
     "type": "speech",
     "speaker": [ "mon_mi_go", "mon_mi_go_slaver", "mon_mi_go_myrmidon" ],
     "sound": "\"Look at this fuckin' shit we're in, man.\"",
-    "volume": 40
+    "volume": 20
   },
   {
     "type": "speech",
     "speaker": [ "mon_mi_go", "mon_mi_go_slaver", "mon_mi_go_myrmidon" ],
     "sound": "\"Every man has got a breaking point.\"",
-    "volume": 40
+    "volume": 20
   },
   {
     "type": "speech",
     "speaker": [ "mon_mi_go", "mon_mi_go_slaver", "mon_mi_go_myrmidon" ],
     "sound": "\"I'ma cut those fuckin' tentacles off, bitch!\"",
-    "volume": 30
+    "volume": 32
   },
   {
     "type": "speech",
     "speaker": [ "mon_mi_go", "mon_mi_go_slaver", "mon_mi_go_myrmidon" ],
     "sound": "\"Watch you bleed out!\"",
-    "volume": 40
+    "volume": 20
```

```suggestion
    "volume": 32
```
Judging by the exclamation point.